### PR TITLE
Issue 711 plugin registry

### PR DESCRIPTION
**Pull request by colin-grapl**
### Which issue does this PR correspond to?
711

### What changes does this PR make to Grapl? Why?
Skeleton implementation of the plugin-registry service.

### How were these changes tested?
No additional testing.

### CODE REVIEW

File: src/rust/plugin-registry/src/client.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (christophermaier) [inline src/rust/plugin-registry/src/client.rs:1]: Is this file necessary?

File: src/rust/plugin-registry/src/main.rs [hunk 2]
```diff
@@ -0,0 +1 @@
+fn main() {}
```
Comment (christophermaier) [inline src/rust/plugin-registry/src/main.rs:1]: This could be a more useful stub, along the lines of what's present for the work queue.

File: src/rust/plugin-registry/src/client.rs (hunk 1 above)
Comment (colin-grapl) [reply src/rust/plugin-registry/src/client.rs:1]: Guess I forgot to impl the client lol

**Review by wimax-grapl — APPROVED** [review]
